### DISCUSSION TITLE
Valid HTTPS certificate

### DISCUSSION THREAD

**Discussion by lpossamai** [modoboa/modoboa#2219] [Help] [answered]
Hi all.

When I open my Modoboa dashboard, I get the following error: `NET::ERR_CERT_INVALID`

I installed Modoboa using the [recommended way](https://modoboa.readthedocs.io/en/latest/installation.html#recommended-way), Is it possible to use Let's encrypt to have a valid certificate? And do I need to perform that manually?

**Comment by lpossamai** [comment, marked as answer]
I managed to get Let's Encrypt working by following [this link](https://certbot.eff.org/lets-encrypt/ubuntufocal-nginx). However, was wondering if Modoboa has plans on implementing it automatically?

**Reply by tonioo** [reply]
You have an option in the installer for that.

**Reply by lpossamai** [reply]
Really? Nothing was shown to me. Is that in the documentation?

**Reply by tonioo** [reply]
https://github.com/modoboa/modoboa-installer#lets-encrypt-certificate

**Reply by lpossamai** [reply]
It's not here https://modoboa.readthedocs.io/en/latest/installation.html#recommended-way - I searched for "Let's Encrypt" on that documentation and it didn't show me anything. Perhaps it's worth mentioning and linking it to the URL you provided.
